fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2628,4 +2628,10 @@ public class Config extends ConfigBase {
 
     @ConfField(mutable = true)
     public static boolean use_lock_manager = false;
+
+    /*
+     * The routine load is set as paused after any task of this job has failed for more than the interval.
+     */
+    @ConfField(mutable = true)
+    public static long routine_load_failure_pause_interval_second = 24L * 3600L;
```

Comment:
Continuous failures may not cover all scenarios. For example, if the routine load fails for 90% of tasks due to an oom issue, it can still result in consumption delays. Therefore, we should notify users of the job's abnormal status based on consumption delays and task failure rates. In fact, turning the job status into PAUSE is just one notification method; it may not reasonable.